tests/vm_test.go

```diff
@@ -978,6 +979,49 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 		})
 	})
+
+	Context("with oc/kubectl", func() {
+		var vm *v1.VirtualMachine
+		var err error
+		var vmJson string
+
+		// Get proper commandline client: oc or kubectl
+		k8sClient := tests.GetK8sCmdClient()
+
+		AfterEach(func() {
+			if vmJson != "" {
+				err = os.Remove(vmJson)
+				Expect(err).ToNot(HaveOccurred())
+				vmJson = ""
+			}
+		})
+
+		It("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should create VM only once", func() {
+			vm = tests.NewRandomVMWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+
+			vmJson, err = tests.GenerateVMJson(vm)
+			Expect(err).ToNot(HaveOccurred(), "Cannot generate vmJson")
+
+			By("Creating VM with DataVolumeTemplate entry with k8s client binary")
+			_, _, err = tests.RunCommand(k8sClient, "create", "-f", vmJson)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Verifying VM is created")
+			Eventually(func() string {
+				newVM, err := virtClient.VirtualMachine(vm.Namespace).Get(vm.Name, &v12.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return newVM.Name
+			}, 15*time.Second, 1*time.Second).Should(Equal(vm.Name),
+				"New VM was not created")
+
+			By("Creating the VM again")
+			_, stdErr, err := tests.RunCommand(k8sClient, "create", "-f", vmJson)
+			Expect(err).To(HaveOccurred())
+
+			Expect(strings.HasPrefix(stdErr, "Error from server (AlreadyExists): error when creating")).To(BeTrue())
```

Comment:
If this fails the error is going to be "false != true". Please add an explanation.